nifi-registry-core/nifi-registry-web-ui/src/main/webapp/nf-registry-bootstrap.js

```diff
@@ -54,7 +54,7 @@ const providers = [];
 if (!locale || locale === 'en-us') {
     bootstrapModule();
 } else { //load the translation providers and bootstrap the module
-    var translationFile = './nifi-registry/assets/locale/messages.' + locale + '.xlf';
+    var translationFile = 'assets/locale/messages.' + locale + '.xlf';
```

Comment:
I'm not sure if this was working earlier or not but with these changes this file throws an error of 404.

Reply:
The only available local is `es`.
https://github.com/apache/nifi-registry/tree/master/nifi-registry-core/nifi-registry-web-ui/src/main/locale

I can get 200 result if I set browser language to Spanish. But the application failed with following error. I'll check if this was working before.
![image](https://user-images.githubusercontent.com/1107620/62909540-f5ae0c80-bdb7-11e9-898d-61c573642cee.png)

Reply:
I've filed NIFIREG-303 and its PR is available for review #215 . It'd be great if you can review the change. Thanks!